examples/plotting/magnetogram_active_regions.py

```diff
@@ -13,34 +13,37 @@
 from astropy.time import TimeDelta
 
 import sunpy.coordinates
+import sunpy.data.sample
 import sunpy.map
 from sunpy.io.special import srs
 from sunpy.net import Fido
 from sunpy.net import attrs as a
 from sunpy.time import parse_time
 
 ##############################################################################
-# For this example, we will search for and download a single HMI using Fido.
-start_time = parse_time("2017-01-25")
-end_time = start_time + TimeDelta(23*u.hour + 59*u.minute + 59*u.second)
-results = Fido.search(a.Time(start_time, end_time),
-                      a.Instrument.hmi & a.Physobs.los_magnetic_field,
-                      a.Sample(60 * u.second))
+# For this example, we will start with the sample data. We need an HMI
+# file and use it to create a map.
+
+smap = sunpy.map.Map(sunpy.data.sample.HMI_LOS_IMAGE)
 
 ##############################################################################
-# Let's select only the first file, download it and create a map.
-result = results[0, 0]
-file_name = Fido.fetch(result)
-smap = sunpy.map.Map(file_name)
+# Then extract the observation time of the sample image. We use it to
+# calculate the time interval for the SRS file search. SRS files are published
+# at the same time each day. Therefore to define `TimeDelta` we use '23*u.hour
+# + 59*u.minute + 59*u.second' interval (not '1*u.day').
```

Comment:
```suggestion
# Then using the observation time of the sample image we will download the corresponding
# NOAA SWPC solar region summary. These are published once a day, so we will define an
# ``end_time`` that ends just before the next day, otherwise you will download two files.
```